cylc/flow/etc/tutorial/cylc-forecasting-workflow/etc/python-job.settings

```diff
@@ -7,6 +7,6 @@
         [[[environment]]]
             # These environment variables ensure that tasks can
             # run in the same environment as the workflow:
-            {% from "sys" import path, executable %}
-            PYTHONPATH = {{':'.join(path)}}
+            {% from "sys" import executable %}
+            PYTHONPATH = {{executable}}/../lib/python:$PYTHONPATH
```

Comment:
Are you sure this works?

```
~/.conda/envs/cylc8/bin/python3.10/../lib: Not a directory
```

Reply:
Try it out! Note, it may behave differently in your interactive shell.

Reply:
I'm not sure setting `PYTHONPATH` is needed? The workflow works for me without it (it seems only the setting of `PATH` below is needed)

Reply:
Why is this back?